src/main/java/io/curity/identityserver/plugin/github/authentication/CallbackRequestHandler.java

```diff
@@ -204,7 +190,8 @@ private WebServiceClient getWebServiceClient(String uri)
 
     private Map<String, Object> redeemCodeForTokens(CallbackGetRequestModel requestModel)
     {
-        HttpResponse tokenResponse = getWebServiceClient("https://github.com/login/oauth/access_token")
+        HttpResponse tokenResponse = getWebServiceClient("https://github.com/")
+                .withPath("login/oauth/access_token")
```

Comment:
I wonder how many plug-ins where broken by this accidental change?

Reply:
This test is passing locally here:
```
def "Can create a request that uses the path provided in the URI create overload"() {
        given:
        def path = "/users"
        URI expectedUri = URI.create("https://github.com" + path)
        WebServiceClientFactory factory = new DefaultWebServiceClientFactory()

        when:
        def actualUri = factory.create(expectedUri).request().get().uri()

        then:
        expectedUri == actualUri
        path == actualUri.path // Should be true if above is true, but URI does funny stuff from time to time so be sure
    }
```
Why was this change needed?

Reply:
The problem was not there, but how we instantiate the WebServiceClient in line 181.
`return _webServiceClientFactory.create(h).withHost(u.getHost());`
the path is not considered. I ll fix that instead